src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -63,7 +63,8 @@ public boolean equals(Object obj) {
         ModelManager other = (ModelManager) obj;
         return addressBook.equals(other.addressBook)
                 && userPrefs.equals(other.userPrefs)
-                && filteredPersons.equals(other.filteredPersons);
+                && filteredPersons.equals(other.filteredPersons)
```

Comment:
Missing check for projectsFolder, but no worries i can correct in my next pr since it's my mistake.